keyboards/pitta/sq80/kb.h

```diff
@@ -0,0 +1,22 @@
+#ifndef KB_H
```

Comment:
This file should be renamed to `sq80.c`.

And if you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch? 

```suggestion
#pragma once
```